README.md

```diff
@@ -1,12 +1,17 @@
-# RunBook
-
-Student Robotics Competition Program
+# Student Robotics Volunteer RunBook
 
 [![CircleCI](https://circleci.com/gh/srobo/runbook.svg?style=svg)](https://circleci.com/gh/srobo/runbook)
 
-# Installation
+The runbook aims to be a single source for all Student Robotics internal
+documentation. This includes information for the [Competition Team][competition-team]
+and [Kit Team][kit-team], as well as other activities.
```

Comment:
The Kit Team Committee are due to discuss where we want to document things in our first meeting.

I agree it would make sense to put it here.

Reply:
Having read the other README file, I am now more unsure about where to document things. The runbook is a place for processes etc, which makes a lot of sense.

Open to suggestions about where we put technical documentation, but I doubt a decision will be made before the full Kit Team is formed.

Reply:
Personally I think I'd expect that detailed technical documentation for the kits would lie within their own source repos where that's possible. If we do _need_ to have that sort of thing for, say, the ODROIDs I think it would be fine for it to be in here (though I'd hope we could work with the upstream in most cases).